app/models/reading.rb

```diff
@@ -14,9 +14,13 @@ def self.update_or_create(work, user)
   end
 
   # called from reading controller
-  def self.mark_to_read_later(work, user)
-    reading_json = [user.id, Time.now, work.id, work.major_version, work.minor_version, true].to_json
-    REDIS_GENERAL.sadd("Reading:new", reading_json)
+  def self.mark_to_read_later(work, user, toread)
```

Comment:
Okay! In that case, this only needs https://github.com/otwcode/otwarchive/pull/2481/files#diff-47cdfed9877703d7f3ff7b154fa65728R99